app/javascript/scavenger_hunt/data/staticData/clues.js

```diff
@@ -1,21 +1,92 @@
 export const Clues = [
   {
-    question: "There’s a student government at UWindsor that represents all undergraduate students in the Faculty of Science. What’s the name of this organization?",
+    puzzleNo: '1',
+    interactive: false,
+    question: "## There’s a student government at UWindsor that represents all undergraduate students in the Faculty of Science. What’s the name of this organization?",
     hint: "",
-    answer: "1"
+    answers: ["Science Society"]
   },
   {
-    question: "What’s Science Society’s website? www.________.com",
+    puzzleNo: '2',
+    interactive: false,
+    question: "## What’s Science Society’s website? www.________.com",
     hint: "",
-    answer: "2"
+    answers: ["uwinscisoc"]
   },
   {
-    question: "Q.3 SciSoc organizes many annual events. In particular, we celebrate two science-related dates every year. What are those days?",
+    puzzleNo: '3',
+    interactive: false,
+    question: "## SciSoc organizes many annual events. In particular, we celebrate two science-related dates every year. What are those days?",
     hint: "Hint: On one day, we give out free guacaMOLE and on the other, it’s free PIe!",
-    answer: "3"
+    answers: ["3"]
```

Comment:
Should be 2 dates in the format ```mm/dd/yyyy```

Reply:
should we use that format rather than `dd/mm/yyyy`?

Reply:
This is temporary. I am not checking the answers right now. I was hoping @prairir will take care of this when we increment the progress?

Reply:
But I can try changing the date formate

Reply:
![image](https://user-images.githubusercontent.com/46072181/92146749-28bea600-ede8-11ea-83ad-5cf9516170ba.png)